ethereum/permissioning/src/test/java/org/hyperledger/besu/ethereum/permissioning/node/NodePermissioningControllerFactoryTest.java

```diff
@@ -55,6 +56,11 @@
   SmartContractPermissioningConfiguration smartContractPermissioningConfiguration;
   PermissioningConfiguration config;
 
+  @Before
+  public void before() {
+    when(transactionSimulator.doesAddressExistAtHead(any())).thenReturn(Optional.of(true));
+  }
+
```

Comment:
is this needed? no other changes in this test

Reply:
It is! because now the existence of the contract is tested a bit earlier in the code. Without this, all tests fail :)